README.md

```diff
@@ -1,27 +1,158 @@
+    border:none;
+    display: table;
+  }
+  table.linktable tr {
+    border: none;
+    background-color: #fff!important;
+  }
+  table.linktable td {
+    border:none;
+  }
+  table.linktable td .win-link {
+    margin-top: 10px;
+  }
+  table.linktable td .mac-link {
+    margin-top: 6px;
+  }
+  table.linktable p {
+    width: 180px;
+    position: relative;
+    bottom: 0;
+  }
+  .disabled:before {
+    content: 'Coming soon';
+    background-color: #9a9797c7;
+    color: white;
+    position: absolute;
+    height: 60px;
+    text-align: center;
+    vertical-align: bottom;
+    line-height: 60px;
+    width: 180px;
+  }
+</style>
+
+<h1 align="center">
+  <a href="https://toggl.com"><img src="https://toggl.com/site/images/media-toolkit/logo_02-644bd26148b73c19d9c91e5baecd8e31.jpg" alt="Toggl" width="200"></a>
+</h1>
+
+<h4 align="center">Native desktop applications for the leading time tracking tool <a href="https://toggl.com" target="_blank">Toggl</a>.</h4>
+
+<p align="center">
+    <a href="https://github.com/toggl/toggldesktop/commits/master">
+    <img src="https://img.shields.io/github/last-commit/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub last commit">
+    <a href="https://github.com/toggl/toggldesktop/issues">
+    <img src="https://img.shields.io/github/issues-raw/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub issues">
+    <a href="https://github.com/toggl/toggldesktop/pulls">
+    <img src="https://img.shields.io/github/issues-pr-raw/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub pull requests">
+    <img src="https://img.shields.io/badge/licence-BSD--3-green"
+         alt="Licence BSD-3">
+</p>
+
+<p align="center">
+  <a href="#features">About</a> •
+  <a href="#download">Download</a> •
+  <a href="#build">Build</a> •
+  <a href="#change-log">Change log</a> •
+  <a href="#contribute">Contribute</a>
+</p>
+
+## About
+
+<table>
+<tr>
+<td>
+
+  **Toggl Desktop** is a Toggl time tracking client with many helper functionas that make tracking time more effortless and smooth. Features such as Idle detection, reminders to track and Pomodoro Timer make this app a great companion when productivity and efficiancy is the goal
+
+</td>
+</tr>
+</table>
+
+<img src="https://user-images.githubusercontent.com/842229/63856838-3a869580-c9ab-11e9-9e36-7db23059ce29.png"
+         alt="Toggl Desktop apps">
+
+## Download
+
+Toggl built and signed apps for all platforms
+
+<table class="linktable">
+  <tr align="center">
+    <td valign="top">
+      <p>Windows</p>
+      <br/>
+      <a class="link" href="https://toggl.github.io/toggldesktop/download/windows-stable/">32bit installer</a>
+      <br/>
+      <a class="link" href="https://toggl.github.io/toggldesktop/download/windows64-stable/">64bit installer</a>
+      <br/>
+      <a class="link" href="https://chocolatey.org/packages/toggl">Chocolatey</a>
+      <br/>
```

Comment:
I would insert some space between Chocolatey link and Store badge as they look cramped. Maybe another linebreak.

Reply:
Apparently, the style wasn't applied, and it should fix such issues.